setup.py

```diff
@@ -25,8 +25,9 @@
 version = "0.4.0"
 release_status = "Development Status :: 4 - Beta"
 dependencies = [
-    "google-api-core[grpc] >= 1.14.0, < 2.0.0dev",
-    'enum34; python_version < "3.4"',
+    "google-api-core[grpc] >= 1.17.2, < 2.0.0dev",
```

Comment:
```suggestion
    "google-api-core[grpc] >= 1.21.0, < 2.0.0dev",
```

providing a `credentials_file` to the channel is only available in the latest release. https://github.com/googleapis/python-api-core/pull/50